src/tools/sdk/index.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Dart SDK
+description: Dart libraries and command-line tools.
+js:
+- url: /tools/sdk/archive/assets/install.js
+  defer: true
+---
+
+The Dart SDK has the libraries and command-line tools that you need to develop
+Dart apps. To learn about other tools you can use for Dart development, see
+[Dart Tools](/tools).
+
+This site's documentation and examples use the
+[{{site.data.pkg-vers.SDK.vers}}][site SDK version]{:.no-automatic-external}
+release of the **Dart SDK**.
+
+## Install the SDK
+
+A package manager can help you easily install and update the Dart SDK.
+Don’t want to use a package manager? Other options are
+[building the SDK from source][] and
+[downloading the SDK as a zip file](sdk/archive).
```

Comment:
sdk/archive -> /tools/sdk/archive (or archive)?

I somehow ended up with the path .../tools/sdk/, and then this took me to .../tools/sdk/sdk/archive, which was a 404.

I wonder (a) why it's possible to have the `/` at the end of the URL (there's a bug about this somewhere) and (b) how many similar possible 404s exist in this site.

Reply:
Stange. I'm not seeing that. I've change the link to /tools/sdk/archive just in case.

Reply:
Ah, the related issue is on site-webdev: https://github.com/dart-lang/site-webdev/issues/1418